src/book/02-system/01-nubots/09-config-script.mdx

```diff
@@ -0,0 +1,65 @@
+---
+section: System
+chapter: NUbots
+title: Configuration and Script System
+description: Introduction to then configuration and script system
+slug: /system/nubots/config-script
+---
+
+## The Configuration System
+
+The NUbots configuration system is based on a hierarchical [YAML](https://yaml.org/) file system.
+
+The hierarchical structure is as follows
+
+```text
+config/
+├── nugus1/
+│   └── ExampleModule.yaml
+├── nugus2/
+│   └── ExampleModule.yaml
+├── robocup/
+│   └── ExampleModule.yaml
+└── ExampleModule.yaml
+```
+
+The file `config/ExampleModule.yaml` must exist for every module. This file defines all of the configuration for the module.
```

Comment:
Must exist for every module that uses configuration. There are many modules that don't need them and don't have them.